src/zenml/cli/stack.py

```diff
@@ -986,11 +986,10 @@ def up_stack() -> None:
 )
 def down_stack(force: bool = False, old_force: bool = False) -> None:
     """Suspends resources of the active stack deployment.
-
     Args:
         force: Deprovisions local resources instead of suspending them.
         old_force: DEPRECATED: Deprovisions local resources instead of
-            suspending them. Use `-y/--yes` instead.
+            suspending them. Use `-f/--force` instead.
```

Comment:
is this the right message or the opposite "Use `-y/--yes` instead."

Reply:
I was confused by this too but Michael mentioned that for stack down specifically, we are going with -f